ReactAndroid/src/main/java/com/facebook/react/uimanager/NativeViewHierarchyOptimizer.java

```diff
@@ -437,7 +437,7 @@ private void transitionLayoutOnlyViewToNativeView(
 
   private static boolean isLayoutOnlyAndCollapsable(@Nullable ReactStylesDiffMap props) {
     if (props == null) {
-      return true;
+      return false;
```

Comment:
How would you go about getting the sticky headers tags in JS? Seems like it would be relatively complex since we can only get them after the view has been rendered using refs. Maybe remapping indexes to offset for null views that will be removed would end up a lot simpler.